Winbook-USC/winbook-app
# Isaac jacques/nonmember editting

To test this out do the following

-Join a project that you are not the owner of
-Leave the project by going to Project Settings
-Click Back button (This should glitch and take you back to the main page so you will have to click back button twice)
-Once your back there should be no post button, no up/down votes, not comment sections, no add a category. And in Prioritization page there should not be the save button or any value update boxes on the right, and the project settings should only show project memmbers

-Go back to home page and scroll down and you should be able to click on any project wihtout having to join it.
-Click on a project and you should be able to see the project content with the same limitations as above
